cmd/createcluster.go

```diff
@@ -191,6 +197,45 @@ func runCreateCluster(w io.Writer, conf clusterConfig) error {
 		}
 	}
 
+	var depositDatas [][]byte
+	for i := 0; i < numDVs; i++ {
+		sk := secrets[i] // Group secret key for this DV
+		pubkey, err := sk.GetPublicKey()
+		if err != nil {
+			return errors.Wrap(err, "secret to pubkey")
+		}
+
+		pk, err := tblsconv.KeyToETH2(pubkey) // Group pubkey
+		if err != nil {
+			return err
+		}
+
+		withdrawalAddr := "0xc0404ed740a69d11201f5ed297c5732f562c6e4e"
```

Comment:
these fields to be replaced by flags in https://github.com/ObolNetwork/charon/pull/556